src/BenchmarkDotNet/Analysers/ZeroMeasurementHelper.cs

```diff
@@ -0,0 +1,17 @@
+using BenchmarkDotNet.Mathematics.StatisticalTesting;
+
+namespace BenchmarkDotNet.Analysers
+{
+    public class ZeroMeasurementHelper
+    {
+        /// <summary>
+        /// Checks distribution against Zero Measurement hypothesis
+        /// </summary>
+        /// <returns>True if measurement is ZeroMeasurement</returns>
+        public static bool CheckZeroMeasurement(double[] results, double threshold)
+        {
+            var sample = new [] { threshold, threshold };
+            return !WelchTest.Instance.IsGreater(results, sample).NullHypothesisIsRejected;
```

Comment:
If we have `Overhead` iterations, it will be much better to compare `Workload` and `Overhead` samples.
If we don't have `Overhead` iterations, we should perform an honest statistical test against fixed value instead of emulating it via the `{ threshold, threshold }` sample.

Reply:
Welch test doesn't allow check against single value.

Reply:
We should implement a one sample t-test. (If you want, I can add the corresponding API for you).

Reply:
Thank you, but I guess I can handle it.